cloud/shared/bin/run.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python3
+
+import argparse
+import os
+import sys
+import importlib
+
+sys.path.append(os.getcwd())
+
+from cloud.shared.bin.lib.config_loader import ConfigLoader
+from cloud.shared.bin.lib.write_tfvars import TfVarWriter
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '--tag', help='Civiform image tag. Required for Setup and Deploy.')
+    parser.add_argument(
+        '--command',
+        help='Command to run. If ommited will validate config and exit.')
+    parser.add_argument(
+        '--config',
+        default='civiform_config.sh',
+        help='Path to CiviForm deployment config file.')
+
+    args = parser.parse_args()
+    if args.tag:
+        os.environ['TF_VAR_image_tag'] = args.tag
+    elif args.command is not None and args.command != 'destroy':
+        exit('--tag is required')
+
+    os.environ['TF_VAR_FILENAME'] = "setup.auto.tfvars"
+    os.environ['BACKEND_VARS_FILENAME'] = 'backend_vars'
+    os.environ['TERRAFORM_PLAN_OUT_FILE'] = 'terraform_plan'
```

Comment:
FWIW it's odd to see these variables here. Would be better to set them only when we call terraform, and maybe pass them just to the call itself, if it's possible, rather than setting global variables. But it's probably requires more work.